presto-main/src/main/java/com/facebook/presto/execution/QueryStats.java

```diff
@@ -73,6 +76,8 @@
     private final DataSize outputDataSize;
     private final long outputPositions;
 
+    private final List<OperatorStats> operatorSummaries;
```

Comment:
Yes, this is fixed (we simply don't return these to coordinator until the task is completed)